prow/cluster/starter.yaml

```diff
@@ -0,0 +1,247 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: plugins
+data:
+  plugins:
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config
+data:
+  config: |
+    prowjob_namespace: default
+    pod_namespace: default
+    sinker:
+      resync_period: 1h
+      max_prowjob_age: 24h
+      max_pod_age: 12h
+    periodics:
+    - interval: 10m
+      agent: kubernetes
+      name: echo-test
+      spec:
+        containers:
+        - image: alpine
+          command: ["/bin/date"]
+---
+apiVersion: extensions/v1beta1
+kind: ThirdPartyResource
```

Comment:
Add directly a CRD? We use one [here](https://github.com/openshift/release/blob/master/cluster/ci/config/prow/prow_crd.yaml).

Reply:
Yes, will do.

Reply:
I'll do this in a follow-up. Need to upgrade my prow cluster to 1.7 before I can test it out...